MAUI/Calendar/getting-started.md

```diff
@@ -0,0 +1,154 @@
+        SfCalendar calendar = new SfCalendar();
+        this.Content = calendar;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+
+{% tabs %}
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 10" %}
+
+using Syncfusion.Maui.Core.Hosting;
+namespace GettingStarted
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+
+            builder.ConfigureSyncfusionCore();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                fonts.AddFont("Segoe-mdl2.ttf", "SegoeMDL2");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Change different calendar views
+
+The .NET MAUI Calendar control provides four different types of views to display dates and it can be assigned to the control by using the `View` property. The control is assigned to the Month view by default. Initially, all Calendar views will show the current date.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.View = CalendarView.Month;
+
+{% endhighlight %}
+{% endtabs %}
+
+![change-different-calendar-views-in-maui-calendar](images/getting-started/maui-calendar-month-view.png)
+
+## Change first day of week
+
+The Calendar control rendered with `Sunday` as first day of week and it  provides customization to change the first day of the week with the FirstDayOfWeek property in month view.
+
+The following code shows the Calendar with `Monday` as the first day of the week.
+
+{% tabs %}  
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar x:Name="calendar">
+    <calendar:SfCalendar.MonthView>
+        <calendar:CalendarMonthView FirstDayOfWeek="Monday"/>
+    </calendar:SfCalendar.MonthView>
+ </calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.MonthView.FirstDayOfWeek = DayOfWeek.Monday;
+
+{% endhighlight %}  
+{% endtabs %}
+
+![change-first-day-of-week-in-maui-calendar](images/getting-started/maui-calendar-first-day-of-week.png)
+
+## Date selection
+
+Calendar allows the user to Select a single date, multiple dates, or a range of dates through interaction and programmatically. The default selection mode is `Default`. 
+
+The selection details can be obtained by using SelectionChanged event of calendar. It returns `CalendarSelectionChangedEventArgs` which holds the details about selected date or range.
```

Comment:
The selection details can be obtained by using the SelectionChanged event of the calendar. It returns `CalendarSelectionChangedEventArgs,` which holds the details about the selected date or range.

Reply:
Content modified.